cypress/integration/chi-vue/search-input.spec.js

```diff
@@ -0,0 +1,216 @@
+}
+
+describe('Search Input', () => {
+  before(() => {
+    cy.visit('tests/chi-vue/search-input.html');
+  });
+
+  describe('Base', () => {
+    it('should exist base search input', () => {
+      cy.get(`${SEARCH_INPUTS_DATA_CY.BASE}`)
+    });
+  });
+
+  describe('Disabled', () => {
+    it('should disable search input', () => {
+      cy.get(`${SEARCH_INPUTS_DATA_CY.DISABLED}`)
+        .find(`${ELEMENTS.INPUT}`)
+        .should('have.attr', 'disabled');
+    });
+  });
+
+  describe('Sizes', () => {
+    const sizes = ['sm', 'md', 'lg', 'xl'];
+
+    sizes.forEach((size) => {
+      it(`should have -${size} size`, () => {
+        checkSizeClass(`[data-cy='${size}-search-input-test']`, `-${size}`);
+      });
+    });
+  });
+
+  describe('Attrs', () => {
+    const attrs = ['name', 'placeholder'];
+    const booleanAttrs = ['portal', 'prevent-value-mutation']
+    const booleanValue = 'true';
+    const value = 'test';
+
+    attrs.forEach((attr) => {
+      it(`should have ${attr} attr`, () => {
+        checkAttr(`${SEARCH_INPUTS_DATA_CY.ATTRS}`, attr, value);
+      });
+    });
+
+    booleanAttrs.forEach((attr) => {
+      it(`should have ${attr} attr at true`, () => {
+        checkBooleanAttr(`${SEARCH_INPUTS_DATA_CY.ATTRS}`, attr, booleanValue);
+      });
+    });
+  });
+
+  describe('Check delete value', () => {
+    const value = 'test';
+    const emptyValue = '';
+
+    it(`should have value`, () => {
+      checkValue(`${SEARCH_INPUTS_DATA_CY.DELETE}`, value);
+    });
+
+    it(`should appear the icon to clean`, () => {
+      cy.get(`${SEARCH_INPUTS_DATA_CY.DELETE}`).find(`${COMPONENT_CLASSES.CLOSE}`);
+    });
+
+    it(`should click to clean`, () => {
+      cy.get(`${SEARCH_INPUTS_DATA_CY.DELETE}`).find(`${COMPONENT_CLASSES.CLOSE}`).click();
+    });
+
+    it(`should have not value`, () => {
+      checkValue(`${SEARCH_INPUTS_DATA_CY.DELETE}`, emptyValue);
+    });
+  });
+
+
+  describe('Events', () => {
+    it(`should trigger chiBlur`, () => {
+      cy.window()
+        .its(`${EXAMPLE_EVENTS.CHI_BLUR}`)
+        .then(chiBlur => {
+          const component = chiBlur.$refs.testEvents;
+          const spy = cy.spy();
+
+          component.$on(`${EVENTS.BLUR}`, spy);
+          cy.get(`${SEARCH_INPUTS_DATA_CY.EVENTS}`).find(`${ELEMENTS.INPUT}`).focus().blur()
+            .then(() => {
+              expect(spy).to.be.called;
+            });
+        });
+    });
+
+    it(`should trigger chiChange`, () => {
+      cy.window()
+        .its(`${EXAMPLE_EVENTS.CHI_CHANGE}`)
+        .then(chiChange => {
+          const component = chiChange.$refs.testEvents;
+          const spy = cy.spy();
+          const value = 'test change';
+
+          component.$on(`${EVENTS.CHANGE}`, spy);
+          cy.get(`${SEARCH_INPUTS_DATA_CY.EVENTS}`).find(`${ELEMENTS.INPUT}`).type(value);
+          cy.get(`${SEARCH_INPUTS_DATA_CY.EVENTS}`).find(`${ELEMENTS.INPUT}`).focus().blur()
```

Comment:
```suggestion
          cy.get(`${SEARCH_INPUTS_DATA_CY.EVENTS}`).find(`${ELEMENTS.INPUT}`).type(value).blur();
```
Not sure if this is going to work but please try shortening the scenario like this

Reply:
It works! Solved.